src/main/java/org/apache/commons/imaging/formats/psd/PsdHeaderInfo.java

```diff
@@ -45,9 +50,14 @@ public PsdHeaderInfo(final int version, final byte[] reserved, final int channel
     }
 
     public void dump() {
-        final PrintWriter pw = new PrintWriter(new OutputStreamWriter(System.out, Charset.defaultCharset()));
-        dump(pw);
-        pw.flush();
+        try (StringWriter sw = new StringWriter(); PrintWriter pw = new PrintWriter(sw)) {
```

Comment:
This seems like duplicate code from a similar block I saw above. Should it be refactored?